.github/workflows/checksums.yml

```
@@ -0,0 +1,64 @@
+name: Compute checksums
+
+on:
+  push:
+    branches: [ checksums ]
+
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'The tag name for which to run it (eg: v22.1.0). If not specified, defaults to the latest release'
+        required: false
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Download binaries
+      shell: bash
+      run: |
+        set -x
+        TAG_NAME=${{ github.event.inputs.tag }}
+        if [ -z "$TAG_NAME" ]; then
+          url=https://api.github.com/repos/NREL/EnergyPlus/releases/latest
+        else
+          url=https://api.github.com/repos/NREL/EnergyPlus/releases/tags/
```

You're right, my bad

```suggestion
          url=https://api.github.com/repos/NREL/EnergyPlus/releases/tags/$TAG_NAME
```